ChaosMod/Effects/EffectsInfo.h

```diff
@@ -506,13 +507,13 @@ const std::unordered_map<EffectType, EffectInfo> g_effectsMap =
 	{EFFECT_PLAYER_KEEP_RUNNING, {"Help My W Key Is Stuck", "player_keeprunning", true}},
 	{EFFECT_VEH_WEAPONS, {"Vehicles Shoot Rockets (Left Click/RB)", "veh_weapons", true}},
 	{EFFECT_MISC_AIRSTRIKE, {"Airstrike Inbound", "misc_airstrike", true}},
-	{EFFECT_FAKE_PLAYER_SWAP, {"Player Swap", "player_playerswap", true, {}, true}},
+	{EFFECT_FAKE_PLAYER_SWAP, {"Player Swap", "player_playerswap", true, {}, true, EffectExecutionType::DEFAULT, "Player Swap"}},
```

Comment:
This seems redundant, however it's the most straight-forward way of the 2 (in my opinion, over `EffectDispatcher::OverrideEffectName`) to get the effect name to show up immediately

Reply:
I'm not sure why it's required for this effect since it doesn't do any faking and doesn't run any `OnStart` logic that prevents the name from showing up